nest/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# ---- BUILD IMAGE ----
+
+FROM node:16-alpine as build
+
+WORKDIR /usr/app
+
+# Install dependencies
+COPY package*.json ./
+RUN npm install
+
+# Compile TypeScript files
+COPY . .
+RUN npm run build
+
+# ---- RUNTIME IMAGE ----
+
+FROM node:16-alpine as runtime
+
+## Set time zone config for Linux image
+ENV TZ="Europe/Oslo"
+
+ARG BUILD_VERSION=unknown
```

Comment:
Can you add a code comment to explain why BUILD_VERSION is set to unknown? It is a bit unclear

Reply:
done :)